sklearn/_fast_gradient_boosting/_loss.pyx

```diff
@@ -0,0 +1,134 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: language_level=3
+
+# Author: Nicolas Hug
+
+cimport cython
+from cython.parallel import prange
+import numpy as np
+cimport numpy as np
+
+from libc.math cimport exp
+
+from .types cimport Y_DTYPE_C
+from .types cimport G_H_DTYPE_C
+
+
+def _update_gradients_least_squares(
```

Comment:
any reason the body of the `..._parallel` functions is not just under these three functions?

Reply:
For some reason I though I couldn't use `prange` in a function that's not `cdef`ed but turns out we can, thanks for catching this